File: PRAKTIKUM MODUL 4/PRAK403-2310817220008-AthayaLailySyafitri.c
#include <stdio.h>

void printCross(int start, int end) {
    for (int i = start; i <= end; i++) {
        printf("%d %d", i, end - i + start);

        if (i < end) {
            printf(" - ");
        }
    }
}

int main() {
    int angka1, angka2;
    scanf("%d %d", &angka1, &angka2);

    if (angka1 > angka2) {
        printCross(angka2, angka1);
    } else if (angka2 > angka1) {
        printCross(angka1, angka2);
    } else {
        printf("%d %d", angka1, angka2);
    }
    return 0;
}

File: PRAKTIKUM MODUL 4/PRAK403-2310817220008-AthayaLailySyafitri.py
def print_cross(start, end):
    for i in range(start, end + 1):
        print(f"{i} {end - i + start}", end="")

        if i < end:
            print(" - ", end="")

def main():
    angka1, angka2 = map(int, input().split())

    if angka1 > angka2:
        print_cross(angka2, angka1)
    elif angka2 > angka1:
        print_cross(angka1, angka2)
    else:
        print(f"{angka1} {angka2}")

if __name__ == "__main__":
    main()